question2_test.py

```diff
@@ -0,0 +1,42 @@
+'''
+Created on 22 May 2017
+
+@author: Eva
+'''
+
+from binary_tree import BinaryTree, TreeNode
+import binary_tree_service
+import unittest
+
+class Question2Test(unittest.TestCase):
+
+    def setUp(self):
+        self.root_node = TreeNode(16)
```

Comment:
It is a bit hard to follow the construction. Something like a comment showing the tree you are building would make it easier to check the tests:
```python
#        16
#       /  \
#      9    18
#     / \     \
#    3   14    19
#   / \
#  1   5
```